doc/rainbow-delimiters.txt

```diff
@@ -172,9 +172,6 @@ available as part of a Lua module.
 The keys are exactly the same as for |g:rainbow_parentheses|.  If fact, this
 function does the same as setting the variable directly.
 
-As an aside, this is a bad practice carried over from a time when Lua support
-in Neovim still had issues with Vim script interoperability, but it has
-persisted through cargo-culting.  You are better off not using this function.
```

Comment:
If this is what @HiPhish still thinks, then I don't think this should be removed.